_data/authors.yaml

```diff
@@ -152,4 +152,10 @@ jclingan:
   job_title: "Senior Principal Product Manager"
   twitter: "jclingan"
   bio: "John Clingan is a Senior Principal Product Manager in Red Hat Middleware focusing on next-generation platforms. He is also an Eclipse MicroProfile co-founder and committer. Prior to Red Hat John spent 8 years as the GlassFish and Java EE product manager at Oracle and Sun Microsystems.  John has 30 years of industry experience (he's kinda old) including software development, technical sales and consulting, and product management. He also has 10+ years of experience teaching Cal State and University of California Extension Java courses to intellectually hungry Java students."
-
+adamevin:
+  name: "Andy Damevin"
+  email: "adamevin@redhat.com"
+  emailhash: "444d54a22db43a0a622180b1ca11dedb"
+  job_title: Senior Software Engineer
+  twitter: "ia3andy"
+  bio: "I am happy to wake up everyday and make the java world a better place with Quarkus. I am mostly a backend developer but I am not that bad at doing web apps either in my spare time, in fact, you can be the judge of that since code.quarkus.io is my baby :-)"
```

Comment:
```suggestion
  bio: "I am happy to wake up every day and make the Java world a better place with Quarkus. I am mostly a backend developer but I am not that bad at doing web apps either in my spare time, in fact, you can be the judge of that since code.quarkus.io is my baby :-)"
```